cli/cmd/dashboard.go

```diff
@@ -44,12 +44,9 @@ var dashboardCmd = &cobra.Command{
 		}
 
 		dashboardAvailable, err := isDashboardAvailable(client)
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed while checking availability of dashboard: %+v\n", err)
-		}
-
-		if !dashboardAvailable {
-			fmt.Fprint(os.Stderr, "Conduit web deployment is not installed in your cluster\n")
+		if err != nil || !dashboardAvailable {
+			fmt.Fprintf(os.Stderr, "Conduit is not running in the \"%s\" namespace\n", controlPlaneNamespace)
+			fmt.Fprintf(os.Stderr, "Install with: conduit install -n %s | kubectl apply -f -\n", controlPlaneNamespace)
```

Comment:
I think this unrelated change should be removed from this commit. If this is fixing a problem then it should be in a commit that describes the problem being fixed. (In particular, I think this *introduces* a problem: We don't report `err` when `err != nil`. If that's intentional then it should be explained in the new PR's commit message too.)

Reply:
This is described in the description for this pull request, as follows:

As part of this change I'm also cleaning up the error message that's printed when conduit is not installed (relates to #289). Previously it would print:

```
$ conduit dashboard -n foo
Failed while checking availability of dashboard: POST to Conduit API endpoint [https://192.168.99.103:8443/api/v1/namespaces/foo/services/http:api:http/proxy/api/v1/SelfCheck] returned HTTP status [404 Not Found]
Conduit web deployment is not installed in your cluster
```

Now it prints:

```
$ conduit dashboard -n foo
Conduit is not running in the "foo" namespace
Install with: conduit install -n foo | kubectl apply -f -
```

Reply:
@briansmith I see your point about discarding the actual error message in favor of the more user-friendly message. I have updated this branch to log the error at debug level, so that it will still be printed if the `--verbose` flag is used.

For instance, without the `--verbose` flag, it prints:

```
$ conduit dashboard
Conduit is not running in the "conduit" namespace
Install with: conduit install -n conduit | kubectl apply -f -
```

But with the `--verbose` flag:

```
$ conduit dashboard --verbose
DEBU[0000] Expecting Conduit Public API to be served over [https://192.168.99.103:8443/api/v1/namespaces/conduit/services/http:api:http/proxy/api/v1/] 
DEBU[0000] Making gRPC-over-HTTP call to [https://192.168.99.103:8443/api/v1/namespaces/conduit/services/http:api:http/proxy/api/v1/SelfCheck] 
DEBU[0000] Response from [https://192.168.99.103:8443/api/v1/namespaces/conduit/services/http:api:http/proxy/api/v1/SelfCheck] had headers: map[Content-Type:[application/json] Content-Length:[182] Date:[Tue, 20 Feb 2018 18:05:14 GMT]] 
DEBU[0000] gRPC-over-HTTP call returned status [404 Not Found] and content length [182] 
DEBU[0000] Error checking dashboard availability: POST to Conduit API endpoint [https://192.168.99.103:8443/api/v1/namespaces/conduit/services/http:api:http/proxy/api/v1/SelfCheck] returned HTTP status [404 Not Found] 
Conduit is not running in the "conduit" namespace
Install with: conduit install -n conduit | kubectl apply -f -
```

That should at least help with debugging when issues are encountered.